swarm/storage/localstore.go

```diff
@@ -97,26 +97,52 @@ func (self *LocalStore) Put(chunk *Chunk) {
 	}
```

Comment:
A bit of godoc for `Put` and its semantics would be appreciated :)

Reply:
Yes, it contains some important logic. Updated.